spec/requests/transaction_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe "Transaction", type: :request do
+  context "when the user's organisation is not associated with the fund" do
```

Comment:
Perhaps this should mention "the user attempts to create a fund for an organisation they do not belong to" ?